src/Form/Form.php

```
@@ -42,6 +42,52 @@ final class Form extends BaseBlock
+     * Returns the array of HTML attributes for the element.
+     *
+     * @return array Attributes array assigned to the element.
+     *
+     * @phpstan-return mixed[]
+     */
+    public function getAttributes(): array
+    {
+        $attributes = parent::getAttributes();
+
+        /** @phpstan-var string|null $id */
+        $id = $this->getAttribute('id', null);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)